resources/views/reports/most_expensive_tags.blade.php

```diff
@@ -17,6 +17,8 @@
                     </div>
                 @endforeach
             </div>
+        @else
+            @include('partials.empty_state', ['payload' => 'most_expensive_tags'])
```

Comment:
If you look in `partials/empty_state.blade.php`, it also shows a link to where you can create the entity that is "empty". 

It's a bit weird having that link for a report, since you can't create a "most expensive tags" report, right?

Reply:
I removed that part in empty_state in a other PR which I will open tomorrow

Reply:
https://github.com/range-of-motion/budget/pull/48